### ISSUE TITLE
unable to 'atomic run' cockpit container after 'atomic install'

### ISSUE THREAD

**Issue by miabbott** [projectatomic/atomic#995]
Using git master (a6d74441ad9980993af4b7168f8dd06632977a5d) on F25 Cloud.

```
# ./atomic images list
   REPOSITORY                               TAG      IMAGE ID       CREATED            VIRTUAL SIZE   TYPE      
   registry.fedoraproject.org/f25/cockpit   latest   40aaf4b46bdb   2017-04-21 02:45   429.17 MB      docker    
   registry.fedoraproject.org/f25/etcd      latest   b0ea91d92c73   2017-05-04 14:57                  ostree    
   registry.fedoraproject.org/f25/flannel   latest   e78f6af84c12   2017-05-05 18:43                  ostree    

# ./atomic install registry.fedoraproject.org/f25/cockpit
/usr/bin/docker run --rm --privileged -v /:/host registry.fedoraproject.org/f25/cockpit /container/atomic-install
+ sed -e /pam_selinux/d -e /pam_sepermit/d /etc/pam.d/cockpit
+ mkdir -p /host/etc/cockpit/ws-certs.d /host/etc/cockpit/machines.d
+ chmod 755 /host/etc/cockpit/ws-certs.d /host/etc/cockpit/machines.d
+ chown root:root /host/etc/cockpit/ws-certs.d /host/etc/cockpit/machines.d
+ mkdir -p /host/var/lib/cockpit
+ chmod 775 /host/var/lib/cockpit
+ chown root:wheel /host/var/lib/cockpit
+ mkdir -p /etc/ssh
+ /bin/mount --bind /host/etc/cockpit /etc/cockpit
+ /usr/sbin/remotectl certificate --ensure

# ./atomic run registry.fedoraproject.org/f25/cockpit                                                                                                                               
The image 'cockpit' appears to have not been installed and has an INSTALL label.  You should install this image first.  Re-run with --ignore to bypass this error.

# ./atomic --debug run registry.fedoraproject.org/f25/cockpit                                                                                                                       
The image 'cockpit' appears to have not been installed and has an INSTALL label.  You should install this image first.  Re-run with --ignore to bypass this error.
Traceback (most recent call last):
  File "./atomic", line 203, in <module>
    sys.exit(_func())
  File "/root/atomic/Atomic/run.py", line 120, in run
    return be.run(img_object, atomic=self, args=self.args)
  File "/root/atomic/Atomic/backends/_docker.py", line 543, in run
    "error.".format(iobject.name or iobject.image))
ValueError: The image 'cockpit' appears to have not been installed and has an INSTALL label.  You should install this image first.  Re-run with --ignore to bypass this error.

# git rev-parse HEAD
a6d74441ad9980993af4b7168f8dd06632977a5d
```

**Comment by miabbott** [comment]
If you append the label to the fully-qualified image name, the `atomic run` command works fine:

```
# atomic run registry.fedoraproject.org/f25/cockpit:latest
/usr/bin/docker run -d --privileged --pid=host -v /:/host registry.fedoraproject.org/f25/cockpit:latest /container/atomic-run --local-ssh

This container uses privileged security switches:

INFO: --pid=host 
      Processes in this container can see and interact with all processes on the host and disables SELinux within the container.

INFO: --privileged 
      This container runs without separation and should be considered the same as root on your system.

For more information on these switches and their security implications, consult the manpage for 'docker run'.

5cb041c8ed6e5504b6d6f0601ec76424a49d2ed7aa830a3ca0b021cfc9983c62
```

**Comment by baude** [comment]
Can we close this?

**Comment by miabbott** [comment]
Closed via #996
